src/detectors/builtin/sendInLoop.ts

```diff
@@ -37,6 +39,10 @@ export class SendInLoop extends ASTDetector {
   severity = Severity.MEDIUM;
 
   async check(cu: CompilationUnit): Promise<MistiTactWarning[]> {
+    const callGraph = new CallGraph(cu.ast);
```

Comment:
Should be removed. We need to put the callgraph in the `CompilationUnit` when building the IR and access it later: `cu.callgraph`.